src/pydata_sphinx_theme/theme/pydata_sphinx_theme/layout.html

```diff
@@ -20,12 +20,108 @@
   <!-- 
     this give us a css class that will be invisible only if js is disabled 
   -->
-  <noscript>
-    <style>
-      .pst-js-only { display: none !important; }
+  <script type="module">
+      import { Octokit, App } from "https://esm.sh/octokit";
+      const octokit = new Octokit();
+      window.octokit = octokit;
+      const anchorTags = document.querySelectorAll('a');
 
-    </style>
-  </noscript>
+      // Regular expression to match the desired GitHub issue URL pattern
+      // the s in pulls/issues is technically wrong but we find it sometimes.
+      const regex = /^https:\/\/github\.com\/([^\/]+)\/([^\/]+)\/(pull|issue)s?\/(\d+)$/;
+
+      // Filter <a> tags that match the pattern
+      const githubIssueLinks = Array.prototype.filter.call(anchorTags, function(element) {
+          return regex.test(element.href);
+      });
+
+      // we can sometime be throttled, or just refused connection,
+      // just do one request on root, and if not 200, just do nothing.
+      const root_res = await octokit.request('GET /');
+      if (root_res.status == 200) {
+
+          githubIssueLinks.map(async (atag) =>  {
+            const match = atag.href.match(regex);
+            if (match) {
+                const [fullUrl, username, repo,pr, issueNumber] = match;
+                console.log("[PST] Username:", username);
+                console.log("[PST] Repository:", repo);
+                console.log("[PST] Issue Number:", issueNumber);
+                try {
+                    // `issues` redirect pull requests as well
+                    const tpl = `GET /repos/${username}/${repo}/issues/${issueNumber}`;
+                    const res = await octokit.request(tpl, {
+                      //owner: 'OWNER',
+                      //repo: 'REPO',
+                      //issue_number: 'ISSUE_NUMBER',
+                      headers: {
+                        'X-GitHub-Api-Version': '2024-09-30'
```

Comment:
This line results in a 400 on the Read the Docs preview build, for example, on [preview -community/topics/attribution](https://pydata-sphinx-theme--1896.org.readthedocs.build/en/1896/community/topics/attribution.html):

"The version you specified in the \"X-GitHub-API-Version\" request header, \"2024-09-30\", is not a supported version. The following versions are currently supported: \"2022-11-28\" (most recent)."